scrapers/MeanBitches.yml

```diff
@@ -0,0 +1,56 @@
+name: MeanBitches
+
+performerByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.meanbitches.com/tour/models/
+    scraper: performerScraper
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - https://www.meanbitches.com/tour/updates/
+    scraper: sceneScraper
+
+
+xPathScrapers:
+  performerScraper:
+    performer:
+      Name: //div[contains(@class,'title_bar')][1]/text()            
+      Image:
+        selector: //tr/td/img[@class="thumbs"]/@src
+        postProcess:
+          - replace:
+            - regex: (.*)
+              with: http://meanbitches.com/$1
+  sceneScraper:
+    common:
+      $titlebar: //table[@id="episode_title"]/tbody/tr/
+    scene:
+      Title: //table[@id="episode_title"]/tbody/tr/td[1]/text()
+      Details: //div[@class="episode_description"]/text()
+      Studio:
+        Name: 
+          fixed: Mean Bitches
+        URL:
+          fixed: https://www.meanbitches.com/
+      Performers:
+        Name: 
+          selector: $titlebar/td[2]/text()
+          postProcess:
+            - replace:
+                - regex: "Featuring:(.*)"
+                  with: $1
```

Comment:
Same here
Also some scenes have multiple performers eg `https://www.meanbitches.com/tour/updates/CuckQuean-Vicki-Chase.html`
```yaml        Name:
          selector: $titlebar/td[contains(comment(),"Featured")]/text()
          postProcess:
            - replace:
                - regex: 'Featuring:\s*(.*)'
                  with: $1
                - regex: '\xa0' # non printable char after the comma
                  with: ""
          split: ","
```
The above seems to work ok in a few scenes i tested